mk/config.mk

```diff
@@ -69,9 +80,9 @@ endif
 
 $(BASE_CONFIGS):
 	@echo "Building target config $@"
-	$(V0) $(MAKE) $(MAKE_PARALLEL) hex CONFIG=$@
+	$(V0) $(MAKE) $(MAKE_PARALLEL) $(OUTPUT_TYPE) CONFIG=$@
```

Comment:
IMO target (and its type) is not available here yet

Reply:
(I assume that this is for config submodule)

PRs will still use older version, but personally I like that - config changes won't break PR